docs/importing_autokey_key_handles.md

```diff
@@ -0,0 +1,118 @@
+# Importing Autokey Key Handles Guidance
+
+If you have any existing [Autokey Key Handles](https://cloud.google.com/kms/docs/resource-hierarchy#key_handles) previously created using [terraform-google-autokey](https://registry.terraform.io/modules/GoogleCloudPlatform/autokey/google) module, it is recommended to import them to [autokey submodule](../modules/autokey/README.md) Terraform state by following the steps below.
+
+**Note:** You don't need to import the existing state for [Autokey configuration](https://cloud.google.com/kms/docs/enable-autokey#enable-autokey-folder) resource. The [autokey submodule](../modules/autokey/README.md) apply process will handle that automatically.
+
+**Note 2:** These instructions were made using [terraform-google-autokey v1.1.1](https://github.com/GoogleCloudPlatform/terraform-google-autokey/releases/tag/v1.1.1) as reference. Future releases versions might require changes in this document.
+
+**WARNING:** [terraform-google-autokey](https://registry.terraform.io/modules/GoogleCloudPlatform/autokey/google) module can be used to create your Autokey folder, Autokey KMS project, Autokey resource project and additional resources (e.g: a Cloud Storage Bucket configured with Autokey), so **DO NOT RUN** a `terraform destroy` for the module, even after the Key Handle import process is completed, unless you just used the module for Autokey configuration and the Key Handle creations only.
+
+## Getting the existing Autokey state from terraform-google-autokey module
+1. Run `cd REPLACE-WITH-YOUR-PATH` to your `terraform-google-autokey/examples/cloud_autokey_example` module path;
+   1. If you didn't use `examples/cloud_autokey_example`, make sure you update the output names below according your terraform files.
+1. Run `terraform output` to get the name of the Autokey folder number and Autokey Key project:
+    ```bash
```

Comment:
Consider using a style guide for your bash.  Can even use a linter if follow a guide 

For example:
 https://google.github.io/styleguide/shellguide.html